src/transaction/log_manager.c

```diff
@@ -1409,6 +1409,24 @@ log_initialize_internal (THREAD_ENTRY * thread_p, const char *db_fullname, const
     }
   log_Gl.rcv_phase = LOG_RESTARTED;
 
+  // PREREQ: metalog is loaded at this point
+  if (is_tran_server_with_remote_storage ())
+    {
+      er_log_debug (ARG_FILE_LINE,
+		    "TEMP: log_initialize_internal: transaction server with remote storage clean shutdown: %s\n",
+		    log_Gl.m_metainfo.get_is_tsrs_shutdown ()? "yes" : "no");
+
```

Comment:
Remove this message; if shutdown was not properly executed and recovery is done, a message should already be logged.
```suggestion
```

Reply:
Done